bash_it.sh

```diff
@@ -3,31 +3,39 @@
 BASH_IT_LOG_PREFIX="core: main: "
 
 # Only set $BASH_IT if it's not already set
-if [ -z "$BASH_IT" ];
-then
-  # Setting $BASH to maintain backwards compatibility
-  export BASH_IT=$BASH
-  BASH="$(bash -c 'echo $BASH')"
-  export BASH
-  BASH_IT_OLD_BASH_SETUP=true
+if [ -z "$BASH_IT" ]; then
+	# Setting $BASH to maintain backwards compatibility
+	export BASH_IT=$BASH
+	BASH="$(bash -c 'echo $BASH')"
+	export BASH
+	BASH_IT_OLD_BASH_SETUP=true
 fi
 
+# Load vendors
 # Load composure first, so we support function metadata
-# shellcheck source=./lib/composure.bash
-source "${BASH_IT}/lib/composure.bash"
+# shellcheck disable=SC1090
+source "${BASH_IT}"/vendor/github.com/erichs/composure/composure.sh
+
 # We need to load logging module first as well in order to be able to log
 # shellcheck source=./lib/log.bash
 source "${BASH_IT}/lib/log.bash"
 
+BASH_IT_LOG_PREFIX="vendor: "
+for _bash_it_vendor_init in "${BASH_IT}"/vendor/init.d/*.bash; do
+	_log_debug "Loading \"$(basename "${_bash_it_vendor_init}" .bash)\"..."
+	# shellcheck disable=SC1090
+	source "${_bash_it_vendor_init}"
+done
+unset _bash_it_vendor_init
+
 # We can only log it now
 [ -z "$BASH_IT_OLD_BASH_SETUP" ] || _log_warning "BASH_IT variable not initialized, please upgrade your bash-it version and reinstall it!"
 
 # For backwards compatibility, look in old BASH_THEME location
-if [ -z "$BASH_IT_THEME" ];
-then
-  _log_warning "BASH_IT_THEME variable not initialized, please upgrade your bash-it version and reinstall it!"
-  export BASH_IT_THEME="$BASH_THEME";
-  unset BASH_THEME;
+if [ -z "$BASH_IT_THEME" ]; then
+	_log_warning "BASH_IT_THEME variable not initialized, please upgrade your bash-it version and reinstall it!"
+	export BASH_IT_THEME="$BASH_THEME"
+	unset BASH_THEME
 fi
 
 # support 'plumbing' metadata
```

Comment:
So, I'd love to add the following line below the first `cite`:
```
cite about-alias about-plugin about-completion
```
This would complete our composure plumbing setup and eliminate the need for every component to call `cite about-<alias,plugin,completion>`, which can be taken care of in follow-up PRs over time.

Reply:
Done, If I understood what you wanted :smile:

Reply:
@buhl  You got it right. Thanks so much!